Userland/Libraries/LibWeb/WebAssembly/WebAssemblyObject.cpp

```diff
@@ -271,8 +256,10 @@ JS_DEFINE_NATIVE_FUNCTION(WebAssemblyObject::instantiate)
                 break;
         }
 
-        if (take_exception_and_reject_if_needed())
-            return promise;
+        if (auto exception = vm.exception()) {
+            vm.clear_exception();
+            return JS::Value(exception);
+        }
```

Comment:
Whatever this is supposed to do, it does something different - there's no `Value(Exception*)` ctor, so you'll probably end up with `Value(bool)`

Reply:
Perhaps looking for `exception->value()`?

Reply:
oh.
god dammit C++